src/Form/Form.php

```diff
@@ -301,7 +276,7 @@ protected static function prepareFieldsForLanguage(
 	/**
 	 * Converts the data of fields to strings
 	 *
-	 * @param false $defaults
+	 * @deprecated 5.0.0 Use `::toStoredValues` instead
 	 */
 	public function strings($defaults = false): array
```

Comment:
`Question:` If we are deprecating this with this PR, should this PR also replace our own using of `::strings()` throughout the core?

Reply:
I think you are right. This is too soon to deprecate it until toStoredValues is a full replacement.